doc/source/building/build_in_buildservice.rst

```diff
@@ -84,11 +84,55 @@ The notable differences to running KIWI locally include:
      choice for multiple package version and results in a much smoother
      experience when using OBS.
 
-* OBS will by default only build a single image type. If your appliance
-  contains multiple build types or uses profiles, use the `multibuild
+* OBS will by default only build a single build type and the default
+  profile. If your appliance uses multiple build types, then you must put
+  each build type into a profile, as OBS cannot handle multiple build
+  types.
+
+  There are two options how to build multiple profiles on OBS. Either, add
```

Comment:
You've started the sentence with "Either", but continue with "or" in line 106. Between these two words, there is too much content. I would try to avoid that.

One way to solve this is to either reduce the content between these words, or try to use a list like this (ha, here is also an either..or example! :wink: ):

```
There are two options how to build multiple profiles on OBS:

1. Use the `<image>` element and add it below the XML declaration (`<?xml ..?>`):

   # add here your code block

1. Use the `multibuild <https://openbuildservice.org/help/manuals/obs-reference-guide/cha.obs.multibuild.html>`_
  feature.
```